### PR TITLE
docs: refa `DonAndDonts` to work without an image

### PR DESCRIPTION
**Pull request by sebald**

### CODE REVIEW

File: docs/content/__internal__/index.mdx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        src="https://upload.wikimedia.org/wikipedia/commons/1/18/Dog_Breeds.jpg"
+        alt="Don't"
+      />
+    </Do.Figure>
+    <Do.Description>
+      Use one primary call to action to help people to proceed
+    </Do.Description>
+  </Do>
+  <Dont>
+    <Do.Figure>
```
Comment (sarahgm) [inline docs/content/__internal__/index.mdx:25]: Shouldn't there be used `Dont.Figure` and `Dont.Description`?

**Review by sarahgm — CHANGES_REQUESTED** [review]

File: docs/content/__internal__/index.mdx (hunk 1 above)
Comment (sarahgm) [reply docs/content/__internal__/index.mdx:25]: is there a difference for them?

Comment (sarahgm) [reply docs/content/__internal__/index.mdx:25]: Can't we just use `Figure` and `Description`?

Comment (sebald) [reply docs/content/__internal__/index.mdx:25]: we could but then hey must be reusable
and yes it should be the dont figure!

Comment (sarahgm) [reply docs/content/__internal__/index.mdx]: I don't understand, you did it accedently wrong and still do.description works for dont component :D

**Review by sarahgm — APPROVED** [review]

**Pull request merged**